src/Microsoft.OData.Edm/Schema/CoreModel/EdmCoreModel.cs

```diff
@@ -31,7 +31,7 @@ public class EdmCoreModel : EdmElement, IEdmModel, IEdmCoreModelElement
 
         private readonly IList<IEdmSchemaElement> coreSchemaElements = new List<IEdmSchemaElement>();
 
-        private readonly IDictionary<string, IEdmSchemaType> coreSchemaTypes = new Dictionary<string, IEdmSchemaType>();
+        private readonly IDictionary<string, IEdmSchemaType> coreSchemaTypes = new Dictionary<string, IEdmSchemaType>(StringComparer.OrdinalIgnoreCase);

```

Comment:
I do think it could be better to have a configuration to enable/disable the case sensitive, otherwise it could be a breaking change.

Reply:
@xuzhg that's a good point. Where would you suggest adding such configuration given that this is in the `EdmCoreModel`?